extensions/metadata_exchange/metadata_exchange.pb.html

```diff
@@ -0,0 +1,67 @@
+---
+title: metadata_exchange
```

Comment:
It seems protoc-gen-docs does not interpret the front matter you put in proto here.